torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10598,6 +10616,18 @@ def ref_pairwise_distance(input1, input2):
                # https://github.com/pytorch/pytorch/issues/58507
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('histc',
+           dtypes=floating_types(),
+           dtypesIfCUDA=floating_types_and(torch.int8, torch.int16, torch.int32, torch.int64),
+           sample_inputs_func=sample_inputs_histc,
+           supports_out=True,
+           supports_autograd=False,
+           skips=(
+               # CUDA histc returns a float tensor but does not correctly warn when passed an integral out tensor
+               # "AssertionError: RuntimeError not raised : Expected RuntimeError when doing an unsafe cast
+               # from a result of dtype torch.float32 into an out= with dtype torch.long"
+               DecorateInfo(unittest.skip("Expected failure!"), 'TestCommon', 'test_out'),
```

Comment:
Just swap this decorator for `unittest.expectedFailure` like this:

https://github.com/pytorch/pytorch/blob/a6c0edff1a60e299d3f9794407f0a1c9a269fd19/torch/testing/_internal/common_methods_invocations.py#L7856